packages/@uppy/vue/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@uppy/vue",
+  "version": "0.0.1",
+  "private": false,
+  "main": "lib/index.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "dev": "parcel src/index.ts  --out-dir lib"
+  },
+  "peerDependencies": {
+    "@uppy/core": "^1.0.0",
+    "vue": ">=2.6.11"
+  },
+  "dependencies": {
+    "shallow-equal": "^1.2.1",
+    "@uppy/dashboard": "^1.12.5",
+    "@uppy/drag-drop": "^1.4.18",
+    "@uppy/progress-bar": "^1.3.18",
+    "@uppy/status-bar": "^1.7.5"
+  },
+  "devDependencies": {
+    "@uppy/core": "^1.0.0",
+    "@uppy/dashboard": "^1.12.5",
+    "@uppy/drag-drop": "^1.4.18",
+    "@uppy/progress-bar": "^1.3.18",
+    "@uppy/status-bar": "^1.7.5",
+    "@vue/component-compiler-utils": "^3.2.0",
+    "parcel-bundler": "^1.12.4",
+    "typescript": "^4.0.3",
+    "vue": "^2.6.12"
+  }
```

Comment:
```suggestion
},
"publishConfig": {
  "access": "public"
}
```

Reply:
Because otherwise npm will try to publish `@uppy/vue` as a private package due to `@` in the name.